shared/common-adapters/list.native.js

```diff
@@ -30,13 +30,7 @@ class List extends PureComponent<Props<any>, void> {
 
   render() {
     return (
-      <View
-        style={{
-          flexGrow: 1,
-          position: 'relative',
-          ...this.props.style,
-        }}
-      >
+      <View style={collapseStyles([styles.outerView, this.props.style])}>
```

Comment:
I was getting errors with spreading `this.props.style` on mobile, so it seemed like a good opportunity to do some minor refactoring using `collapseStyles` and `styleSheetCreate`.

Reply:
👍 we definitely want all the common-adapters to be able to take ids